src/index.js

```diff
@@ -270,10 +270,35 @@ function statelessComponentHook(Ctor) {
 	return Wrapped;
 }
 
+function validatePropTypes (vnode) {
+	if (DEV) {
+		let componentClass = typeof vnode.nodeName === "function"
+			? vnode.nodeName
+			: vnode.type;
+
+		if (typeof componentClass !== "function") return;
+		let name = componentClass.displayName || componentClass.name;
+		let propTypes = componentClass.propTypes;
+		if (propTypes) {
+			let props = vnode.props;
+			if (
+				!(vnode.props && vnode.props.children) &&
+				vnode.children &&
+				vnode.children.length
+			) {
+				props.children = vnode.children;
+			}
+			propsHook(props);
```

Comment:
I think I see what you mean now. 🤔 I don't think so. I could have sworn we fixed this a while back but I can't find where.